solax/inverters/x1_hybrid_gen4.py

```diff
@@ -15,28 +16,20 @@ class X1HybridGen4(Inverter):
                 "sn",
             ): str,
             vol.Required("ver"): str,
-            vol.Required("Data"): vol.Schema(
+            vol.Required("data"): vol.Schema(
```

Comment:
It is `Data`, not `data` in JSON. Just like `Information` below. Why did you change it?

Reply:
Because I lowercased all keys in a previous PR.

Reply:
I accedentilly hit "Comment" in the mobile app...

I meant to write `commit` not `PR`, but both are wrong, [it was in the same commit](https://github.com/squishykid/solax/pull/145/commits/4c440632e5ba9cf9ebf6403708453c4cbca1214c#diff-0b94a8b014d1902625d8bd9f0ef7e94be86fd081927dd26e0c74e777572bfd59R109).

Anyway, I assume https://github.com/squishykid/solax/issues/161 is why you decided to comment.

Reply:
Yes, I was going through recent changes to see if something broke, but then remembered about firmware upgrade and found the root cause